template/partial/impact_of_assault/impact_3.html

```diff
@@ -0,0 +1,22 @@
+<table class="ui large very padded definition table">
+    <tbody>
+      <tr>
+        <td class="three wide dark-orange">Psychological/ Emotional</td>
+      </tr>
+      <tr>
+		<td class="dark-green">
+	        <ul>
+	            <li>Depression</li>
+				<li>Anger</li>
+				<li>Fear</li>
+				<li>Anxiety</li>
+				<li>Guilt</li>
+				<li>Shock</li>
+				<li>Shame/ Embarrassment</li>
```

Comment:
#### This is again a github issue, Please check it in local. Indentation is proper.
<img width="1280" alt="screen shot 2017-06-16 at 13 39 28" src="https://user-images.githubusercontent.com/13470643/27217853-3c8522b0-5299-11e7-9036-a68bd4cb08ac.png">